flixel/system/frontEnds/HTML5FrontEnd.hx

```diff
@@ -70,4 +128,17 @@ enum FlxBrowser
 	OPERA;
 	UNKNOWN;
 }
+
+enum FlxPlatform
+{
+	WINDOWS;
+	LINUX;
+	UNIX;
```

Comment:
What exactly does `UNIX` mean in this context? As I understand, both Linux and Mac are technically UNIX systems. That wouldn't make much sense in an enum though.